src/state/compoundSelectors/index.ts

```diff
@@ -27,3 +28,17 @@ export const getCurrentUIData = createSelector(
             : defaultData;
     }
 );
+
+export const getDefaultUISettingsApplied = createSelector(
```

Comment:
Consider changing to `getIsDefaultUISettingsApplied` (or `Are`) so this sounds more like a boolean/conditional value. Otherwise `getDefaultUISettingsApplied` sounds like it would be returning the default UI settings.